model/src/main/proto/interaction_object.proto

```diff
@@ -49,3 +50,19 @@ message Fraction {
   int32 denominator = 4;
 }
 
+// Structure for a ListOfString object.
+message ListOfSetsOfHtmlStrings {
+  repeated StringList list = 1;
```

Comment:
Lets maybe call this set_of_html_strings instead of list?

Reply:
Done

Reply:
This doesn't seem to be done? What I meant is change
`repeated StringList list = 1` to 
`repeated StringList set_of_html_strings = 1`

Reply:
Was an internet issue

Reply:
Ah okay got it , thanks! This now LGTM